src/TLuaInterpreter.cpp

```diff
@@ -3224,6 +3224,11 @@ int TLuaInterpreter::setFont(lua_State* L)
     }
     QString font{lua_tostring(L, s)};
 
+    if (!mudlet::self()->getAvailableFonts().contains(font)) {
```

Comment:
Make it a nil+error because existing scripts will start breaking with this change: remember an error in Lua means no further of the script is run, and a Mudlet update that breaks your scripts isn't good

Reply:
Currently setting fonts is case-insensitive and I suspect this changes the behaviour to be case-sensitive, can you check?

Reply:
Yes it will be case sensitive, will try to work around.